src/CSFML/Audio/Music.cpp

```diff
@@ -90,6 +94,27 @@ bool sfMusic_isLooping(const sfMusic* music)
 }
 
 
+////////////////////////////////////////////////////////////
+void sfMusic_setEffectProcessor(sfMusic* music, sfEffectProcessor effectProcessor)
+{
+    assert(music);
+    if (!effectProcessor)
+    {
+        music->This.setEffectProcessor(nullptr);
+    }
+    else
+    {
+        music->This.setEffectProcessor(
+            [=](const float*  inputFrames,
```

Comment:
```suggestion
            [effectProcessor](const float*  inputFrames,
```
When lambdas only capture one of two things I prefer we do so explicitly.